DB/postgresCRUD.js

```diff
@@ -1,84 +1,141 @@
-const { Sequelize, DataTypes } = require('sequelize');
-
-const sequelize = new Sequelize('test', 'postgres', 'hrr48', {
-  dialect: 'postgres',
-  storage: 'localhost:5432',
-  logging: false
-})
-
-
-const test = async () => {
-  try {
-    await sequelize.authenticate();
-    console.log('Connection has been established successfully.');
-  } catch (error) {
-    console.error('Unable to connect to the database:', error);
-  }
+const { Pool } = require('pg');
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'carousel',
+  password: 'hrr48',
```

Comment:
make sure you .gitignore DB passwords.  use a .env file or a .cfg file and put that file in your .gitignore.